packages/lib-user/src/components/shared/Select/Select.js

```diff
@@ -0,0 +1,65 @@
+import { Box, Select as GrommetSelect, ThemeContext } from 'grommet'
+import { arrayOf, func, shape, string } from 'prop-types'
+import { useEffect, useState } from 'react'
+
+import selectTheme from './theme'
+
+const DEFAULT_HANDLER = () => {}
+const DEFAULT_VALUE = { label: '', value: '' }
+
+function Select({
+  id = '',
+  name = '',
+  handleChange = DEFAULT_HANDLER,
+  options = [],
+  value = DEFAULT_VALUE
+}) {
+  const [selected, setSelected] = useState(value)
+
+  useEffect(() => {
+    setSelected(value)
+  }, [value])
+
+  function handleSelect(option) {
+    setSelected(option)
+    handleChange(option)
+  }
+
+  return (
+    <ThemeContext.Extend value={selectTheme}>
+      <Box
+        width={{ max: '215px' }}
+      >
+        <GrommetSelect
+          a11yTitle={name}
+          id={id}
+          name={name}
+          labelKey='label'
+          onChange={({ option }) => handleSelect(option)}
+          options={options}
+          size='medium'
+          style={{ textAlign: 'center' }}
```

Comment:
Not sure how I feel about this, but creating a styled-component just to add this one line of css didn't seem ideal either. I tried to add to the custom theme without luck.

Reply:
I think it's ok. We've been making a lot of spacing css inline and when I did the SimpleDropdown project which also uses the Grommet Select I noticed a lot of limitations on customization / stylability.